### PR TITLE
fix: typos in models

### PR DESCRIPTION
**Pull request by EnricoBaivo**
event.py Optional[Achievement] = None # typo achievment  
common.py Achievement(BaseModel) 

- description: str # typo desciption
- mode: Optional[Gamemode] = Null

Example for Achievement that has a null as gamemode
```
"achievement": {
            "id": 319,
            "name": "Autocreation",
            "slug": "all-secret-autocreation",
            "description": "Absolute rule.",
            "grouping": "Hush-Hush",
            "icon_url": "https://assets.ppy.sh/medals/web/all-secret-autocreation.png",
            "mode": null,
            "ordering": 2,
            "instructions": "<i>One is all.</i>"
}
```

## Self-check

- [ ] The changes are tested with [pre-commit](https://pre-commit.com/)
- [ ] The changes pass unit testing with [pytest](https://pre-commit.com/)
- [ ] The changes pass [mypy](http://mypy-lang.org/)

### CODE REVIEW

File: aiosu/models/event.py [hunk 1]
```diff
@@ -65,7 +65,7 @@ class Event(BaseModel):
     type: EventType
     r"""Information on types: https://github.com/ppy/osu-web/blob/master/resources/assets/lib/interfaces/event-json.ts"""
     parse_error: Optional[bool] = None
-    achievment: Optional[Achievement] = None
+    achievement: Optional[Achievement] = None  # typo achievment
```
Comment (NiceAesth) [inline aiosu/models/event.py:68]: Please do not add comments like these to the code

File: aiosu/models/common.py [hunk 2]
```diff
@@ -46,10 +46,10 @@ class Achievement(BaseModel):
     id: int
     name: str
     slug: str
-    desciption: str
+    description: str  # typo  desciption
```
Comment (NiceAesth) [inline aiosu/models/common.py:49]: Same as above

File: .pre-commit-config.yaml [hunk 3]
```diff
@@ -42,4 +42,4 @@ repos:
     -   id: poetry-check
 
 default_language_version:
-    python: python3.9
+    python: python3.11
```
Comment (NiceAesth) [inline .pre-commit-config.yaml:45]: Do not do unrelated changes in prs

File: .gitignore [hunk 4]
```diff
@@ -26,7 +26,8 @@ share/python-wheels/
 .installed.cfg
 *.egg
 MANIFEST
-
+.gitigonre
```
Comment (NiceAesth) [inline .gitignore:29]: Do not do unrelated changes in prs

**Review by NiceAesth — CHANGES_REQUESTED** [review]

**Comment by NiceAesth** [comment]
Could be worth cross checking types with osu-web. https://github.com/ppy/osu-web/tree/master/app/Models

**Review by NiceAesth — APPROVED** [review]

**Comment by NiceAesth** [comment]
I went ahead and fixed the issues myself. Read through the changes I have made in case you wish to see the direction to take PRs in the future.

(also, as a future note I recommend reading https://towardsdatascience.com/why-good-codes-dont-need-comments-92f58de19ad2)

**Comment by EnricoBaivo** [comment]
Thanks alot for the informations, indeed it was my first pull request ever. So im looking forward to learn the correct patterns and rules for prs.

**Pull request merged**